packages/app-staking/src/Overview/Address.tsx

```diff
@@ -37,29 +47,70 @@ class Address extends React.PureComponent<Props> {
       : defaultName;
   }
 
+  private onClickBadge = (): void => {
+    const { badgeExpanded } = this.state;
+    this.setState({ badgeExpanded: !badgeExpanded });
```

Comment:
(nitpick) Prefer a blank line separating blocks, i.e. just a single space after the consts. (And just a nitpick, it doesn't aid functionality, however just makes it _slightly_ easier to read and find patterns)